src/client/apps/edit/components/admin/index.jsx

```diff
@@ -55,12 +66,13 @@ export class EditAdmin extends Component {
 
           <AdminFeaturing />
 
-          <AdminAppearances />
-
-          {isEditorial &&
+          {!isNews &&
+            <AdminAppearances />
+          }
+          {isEditorial && !isNews &&
             <AdminSuperArticle />
           }
-          {isEditorial &&
+          {isEditorial && !isNews &&
             <AdminSponsor />
           }
```

Comment:
Can these cases be collapsed together?

```javascript 
{isEditorial && !isNews &&
  <Fragment>
    <AdminSuperArticle />
    <AdminSponsor />
  </Fragment>   
}
```

Reply:
That should work, but not too sure; might want to investigate since fragments are new.

Reply:
Gonna merge without and keep this in mind for the future.